src/test/cli/metrics/issue-count-test.ts

```diff
@@ -1,9 +1,13 @@
-import test from 'ava';
+import anyTest, {TestInterface} from 'ava';
+import {Server} from 'http';
 
 import {getIssueCounts} from '../../../cli/metrics/issue-counts';
 import {startTestReplayServer} from '../../../replay-server';
 import {initGithub} from '../../../utils/github';
 
+// tslint:disable-next-line:no-any
+const test: TestInterface<{server: Server}> = anyTest as any;
```

Comment:
can you add a comment on what `anyTest` is.

Reply:
I don't think it's anything actually. Its just the default export and I think this line just recasts it to have the right context types.